src/components/Charts/AreaChartTokenPrice.stories.mdx

```diff
@@ -0,0 +1,116 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+import { DAI } from '@swapr/sdk'
+import { Flex } from 'rebass'
+
+import { AreaChartTokenPrice } from './AreaChartTokenPrice'
+
+<Meta title="Charts/AreaChartTokenPrice" component={AreaChartTokenPrice} />
+
+# AreaChartTokenPrice
+
+Area chart token price is just a wrapper on [lightweight-chart](https://github.com/tradingview/lightweight-charts) by Trading view with our custom theme and nice price on hover feature.
+
+<Canvas>
+  <Story name="AreaChartTokenPrice">
+    <Flex bg="#1e1e55" width={['100%', '550px', '550px', '600px', '650px']}>
+      <AreaChartTokenPrice
+        showHours={false}
+        tokenSymbol={DAI[1].symbol}
+        data={[
+          {
+            time: 1662486000,
+            value: '1601.794376003465471685150261540348',
+          },
+          {
+            time: 1662490200,
+            value: '1578.683769390060178475980473812557',
+          },
+          {
+            time: 1662491100,
+            value: '1575.762172192810446717267291093502',
```

Comment:
It would be good to move this data is a sepearate variable since when viewing code it messed up view abit
![Screenshot 2022-09-21 at 19 01 33](https://user-images.githubusercontent.com/33226956/191566688-a0a52fe6-a8a3-4265-aec0-4cbc9a7ba8f2.png)

Reply:
true, I'll move it.

Reply:
Done! tks